GFDL_tools/fv_ada_nudge.F90

```diff
@@ -2516,7 +2514,21 @@ subroutine fv_ada_nudge_end
 
 #ifdef ENABLE_ADA ! snz
 
-    call save_restart(ada_driver_restart)
+    if (open_file(ada_driver_restart, restart_file, "overwrite", domain, is_restart=.true.) then
+       call fv_io_register_axis(ada_driver_restart, numx=1, numy=1, numz=1, zsize=(/size(Atm_var%u_adj,3)/))
```

Comment:
Why use fv_io_register axis here, but in other cases you use the direct call to register_axis?

Reply:
I tried to use fv_io_register_axis when I could to limit the amount of duplicate code needed.  I can go through and ensure that I use the fv_io_register_axis wherever possible.